frontend/src/index.tsx

```diff
@@ -1,11 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {
+  ApolloProvider,
+  ApolloClient,
+  createHttpLink,
+  InMemoryCache,
+} from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+import { REACT_APP_CROWNLABS_GRAPHQL_URL } from './env';
 import './theming';
 import App from './App';
 
+const TOKEN = '';
+
+const httpLink = createHttpLink({
+  uri: REACT_APP_CROWNLABS_GRAPHQL_URL,
+});
+
+const authLink = setContext((_, { headers }) => {
+  const token = TOKEN;
+  return {
+    headers: {
+      ...headers,
+      authorization: token ? `Bearer ${token}` : '',
+    },
+  };
+});
+
+const client = new ApolloClient({
+  link: authLink.concat(httpLink),
+  cache: new InMemoryCache(),
+});
```

Comment:
I would personally move this in its own file to maintain the index clean, but I am unsure whether it is a good practice.

Reply:
@GabriFila  WDYT? 
To be honest, in the official [doc](https://www.apollographql.com/docs/react/networking/advanced-http-networking/) there is no indication of where is the best place to insert them.
But following this [guide](https://www.howtographql.com/react-apollo/1-getting-started/) I have seen that all materials are into the `index.js`.